### ISSUE TITLE
various command expressions

### ISSUE THREAD

**Issue by MrScopes** [SkriptLang/Skript#3553]
```
[the] permission of [command] %string%
[the] permission message of [command] %string%
[the] description of [command] %string%
[the] aliases of [command] %string%
[the] cooldown of [command] %string%
[the] cooldown message of [command] %string%
[the] arguments of [command] %string%
 [the] executable[by] of [command] %string% 
 who can execute [command] %string%
[the] usage [message] of [command] %string%
```
It would be cool if we could use these, they would work for every skript command (non skript-only features should work with external commands if possible)

**Comment by ShaneBeee** [comment]
Sounds like a good idea to me!

**Comment by FranKusmiruk** [comment]
Implemented in #3234
